definitions/tools/check_strand.cwl

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: CommandLineTool
+label: "how_are_we_stranded_here"
+baseCommand: ['check_strandedness']
+arguments: [
+    "--print_commands"
+]
+
+stdout: "$(inputs.reads1.nameroot).strandness_check.txt"
+
+requirements:
+    - class: ResourceRequirement
+      ramMin: 16000
+      tmpdirMin: 25000
+      coresMin: 4
```

Comment:
Does this command use four cores by default?  I can't find any info on multithreading for this tool, but it looks kind of like these requirements may've been copied verbatim from `trim_fastq.cwl`.

Reply:
No that was something that I adapted from another cwl file and did not bother to change. It might not require multi-core